plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DriverConnectionFactory.java

```diff
@@ -34,14 +34,14 @@ public class DriverConnectionFactory
     private final Driver driver;
```

Comment:
When `String` is not ok for value?

Reply:
In case of Trino JDBC driver with constrained delegation we could pass GssCredential as a JDBC property - `KerberosConstrainedDelegation`

Reply:
Because of this https://github.com/trinodb/trino/pull/17853 right?

Reply:
But that is a different thing. Do they need to be in sync?

Reply:
It's not about that specifically. Technically JDBC drivers take `Properties` which is a `Map<String, Object>` so we shouldn't pretend it's `Map<String, String>` to disallow cases where `Object` is actually needed.